CMakeLists.txt

```diff
@@ -33,21 +32,21 @@ add_library(pointcloud_filters src/pointcloud_filters.cpp)
 target_link_libraries(pointcloud_filters ${catkin_LIBRARIES})
 
 add_library(laser_scan_filters src/laser_scan_filters.cpp src/median_filter.cpp src/array_filter.cpp src/box_filter.cpp)
-target_link_libraries(laser_scan_filters ${catkin_LIBRARIES} ${Boost_LIBRARIES} ${Eigen_LIBRARIES})
+target_link_libraries(laser_scan_filters ${catkin_LIBRARIES})
```

Comment:
The "DEPENDS Boost" doesn't appear to append the Boost library flags to `catkin_LIBRARIES` for the current package. @dirk-thomas can you confirm that this is the case?